variants/Generic_F401Rx/variant.h

```diff
@@ -0,0 +1,148 @@
+#define PB10 26  // | 26      |        |           | TWI2_SCL | SPI2_SCK               |           |
+#define PB12 27  // | 27      |        |           |          | SPI2_SS                |           |
+#define PB13 28  // | 28      |        |           |          | SPI2_SCK               |           |
+#define PB14 29  // | 29      |        |           |          | SPI2_MISO              |           |
+#define PB15 30  // | 30      |        |           |          | SPI2_MOSI              |           |
+//                  |---------|--------|-----------|----------|------------------------|-----------|
+#define PC0  A10 // | 31      | A10    |           |          |                        |           |
+#define PC1  A11 // | 32      | A11    |           |          |                        |           |
+#define PC2  A12 // | 33      | A12    |           |          | SPI2_MISO              |           |
+#define PC3  A13 // | 34      | A13    |           |          | SPI2_MOSI              |           |
+#define PC4  A14 // | 35      | A14    |           |          |                        |           |
+#define PC5  A15 // | 36      | A15    |           |          |                        |           |
+#define PC6  37  // | 37      |        | USART6_TX |          |                        |           |
+#define PC7  38  // | 38      |        | USART6_RX |          |                        |           |
+#define PC8  39  // | 39      |        |           |          |                        |           |
+#define PC9  40  // | 40      |        |           | TWI3_SDA |                        |           |
+#define PC10 41  // | 41      |        |           |          | SPI3_SCK               |           |
+#define PC11 42  // | 42      |        |           |          | SPI3_MISO              |           |
+#define PC12 43  // | 43      |        |           |          | SPI3_MOSI              |           |
+#define PC13 44  // | 44      |        |           |          |                        |           |
+#define PC14 45  // | 45      |        |           |          |                        | OSC32_IN  |
+#define PC15 46  // | 46      |        |           |          |                        | OSC32_OUT |
+//                  |---------|--------|-----------|----------|------------------------|-----------|
+#define PD2  47  // | 47      |        |           |          |                        |           |
+//                  |---------|--------|-----------|----------|------------------------|-----------|
+#define PH0  48  // | 48      |        |           |          |                        | OSC_IN    |
+#define PH1  49  // | 49      |        |           |          |                        | OSC_OUT   |
+//                  |---------|--------|-----------|----------|------------------------|-----------|
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        34
```

Comment:
```suggestion
#define NUM_DIGITAL_PINS        50
```

Reply:
But you did this with the other pinouts?

Reply:
The other I modified, I removed the duplicated pins that's all. All pins are digital so in your variants it is 50.